q2cli/cli.py

```diff
@@ -6,30 +6,35 @@
 # The full license is in the file COPYING.txt, distributed with this software.
 # ----------------------------------------------------------------------------
 
-import collections
 import os
+import collections
 
 import click
 import qiime
 import qiime.plugin
 import qiime.sdk
 from qiime.sdk import PluginManager
 
-from . import __version__ as q2cli_version
-
-
-PLUGIN_MANAGER = PluginManager()
+import q2cli._tools
+import q2cli._info
 
 
 class QiimeCLI(click.MultiCommand):
 
+    _plugin_manager = PluginManager()
+    _builtin_commands = {'tools': q2cli._tools.tools,
+                         'info': q2cli._info.info}
+
     def list_commands(self, ctx):
-        plugins = PLUGIN_MANAGER.plugins.keys()
-        return sorted(plugins)
+        plugins = list(sorted(self._plugin_manager.plugins.keys()))
+        commands = list(sorted(self._builtin_commands.keys())) + plugins
+        return commands
```

Comment:
The `+ plugins` is kind of hiding at the end, what if you did:
`return builtins + plugins` which would be more obvious.